account-kit/smart-contracts/package.json

```diff
@@ -35,7 +35,7 @@
     "./package.json": "./package.json"
   },
   "scripts": {
-    "generate": "npx tsx ../plugingen/src/cli.ts generate && yarn _lint:generated",
+    "generate": "npx tsx ../plugingen/src/cli.ts generate && yarn _lint:generated 1> /dev/null",
```

Comment:
piping the stdout lint logs to null because they impact caching and they shouldn't be considered in the diffing logic